modules/monitoring-setting-scrape-sample-and-label-limits-for-user-defined-projects.adoc

```diff
@@ -3,14 +3,15 @@
 // * monitoring/configuring-the-monitoring-stack.adoc
 
 :_content-type: PROCEDURE
-[id="setting-a-scrape-sample-limit-for-user-defined-projects_{context}"]
-= Setting a scrape sample limit for user-defined projects
+[id="setting-scrape-sample-and-label-limits-for-user-defined-projects_{context}"]
+= Setting scrape sample and label limits for user-defined projects
 
 You can limit the number of samples that can be accepted per target scrape in user-defined projects.
+You can also limit the number of scraped labels, the length of the label name, and the length of the label value.
```

Comment:
```suggestion
You can also limit the number of scraped labels, the length of label names, and the length of label values.
```

Reply:
fixed here and in two other places as well